### ISSUE TITLE
Add text tool

### ISSUE THREAD

**Issue by AmoebeLabs** [AmoebeLabs/swiss-army-knife-card#27]
## The Problem To Be Solved
There is no way yet to display random text. It was in the flexible horeseshoe card.
So add this tool and see what functionality can be added to this tool to make it nice:
- text along path
- styling (default)

## (Optional): Suggested Solution
Look here, https://www.hongkiat.com/blog/scalable-vector-graphics-text/ for examples:
- Basic Implementation (maybe using multiple lines??, ie as an array? But how about styling then??)
- Basic Text Styles (bold, italic, etc)
- Writing Mode (horizontal, vertical)
- Text Outline (yep, using `stroke` and setting `fill` to `none`)
- Text Path (text along svg path)
- Text Gradient (requires using a def)

Shouldsupport inline styling also, with `<tspan>` elements as implementation? 
And of course: the delta y, to move the next line below the current line. Always in em units.

```yaml
tool:
  type: text
  lines:
    line1:
      - content: "this is a test"
    line2
      - dy: 1.2
      - content: "another line"
      - content: "and this is on the same line"
      - content: ", but how would I style each part of this line???"
    line3
      - path: "d = ....."
      - content: "this is a test on a path"
  styles:
    lines:
      default:
        font: xyza;
      line1: 
        font: xyza;
      line2:
        font: xyzb;
      line3:
        font: xyzc;
        text-size: 845739;
        <for second part of the line ??????? How ???????>
```
In other words: a line is always an array of tspans. Styling works also: use an array. This conflicts with other tools: styling is NOT an array, but a map. Is inconsistent. Maybe things like `line1.1`, `line1.2` for different lines of content??
